src/signal-forms/signal-input-error.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  AfterViewInit,
+  Directive,
+  inject, InjectionToken, Injector,
+  ViewContainerRef
+} from '@angular/core';
+import {SIGNAL_INPUT_ERROR_COMPONENT} from "@signal-form/signal-input-error.token";
+import {SignalInputDirective} from "@signal-form/signal-input.directive";
+import {FormField} from "@signal-form/form-field";
```

Comment:
The `@signal-form` should only be used in the example app.

```suggestion
import {SIGNAL_INPUT_ERROR_COMPONENT} from "./signal-input-error.token";
import {SignalInputDirective} from "./signal-input.directive";
import {FormField} from "./form-field";
```